woocommerce/woocommerce
# Removes the star icon font

### All Submissions:


### Changes proposed in this Pull Request:

This Pr will replace the WooCommerce font with a new one that is the merge of the previous WooCommerce Font and the Star font. In this way we need to load only a single font to get the same result. 

Closes #31490 

### How to test the changes in this Pull Request:

1. Rebuild assets (this is mandatory because i've changed _font.scss and the other theme related )
2. To know if the patch is working delete all the Star.* fonts from /woocommerce/assets/fonts/star.woff, if the star icon doesn't disappear the font family was correcty replaced. rel to [_fonts.scss](https://github.com/erikyo/woocommerce/blob/remove-star-icon/plugins/woocommerce/legacy/css/_fonts.scss)
3. Check in DevTools under the network panel, filtering fonts. Star.* should not be present
4. While testing .svg font remember that are allowed only before chrome 38, and actually works only in Safari. [source](https://caniuse.com/svg-fonts). The reason was a poor rendering quality, and in addition, some features are missing eg. ligatures. Same for the eot font that can be tested only with IE <= 11


### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Removes star font

#### Comment 

</details>


#### Testing Instructions

1. Begin by setting up a fresh WP/WooCommerce installation, followed by modifying the theme you have selected.
2. Access the font files from [this location](https://github.com/woocommerce/woocommerce/tree/ce03a12790c526e88e382d62dc170b63db7a0884/plugins/woocommerce/assets/fonts) and store them within a folder titled "fonts." Integrate this folder into the root directory of your active theme.
3. Integrate the following code into your `function.php` file:
```php
function add_a_font() {
?>
  <style>
  @font-face {
      font-family: 'new-star';
      src: url('fonts/WooCommerce.woff2') format('woff2'),
      url('fonts/WooCommerce.woff') format('woff'),
      url('fonts/WooCommerce.ttf') format('truetype');
      font-weight: normal;
      font-style: normal;
  }
  </style>
<?php 
}
add_action( 'wp_head', 'add_a_font' );
```
4. Proceed to your website and, using developer tools, inspect a WooCommerce page where the star font is displayed. This may vary depending on your chosen theme, but typically, many themes showcase this font within the review tab of the product page. Once you've located the "star" on your page, toggle the "new-star" font using the developer tools. Verify that there are no discernible changes when adding or removing the "new-star" font. (For thoroughness, please ensure your browser is zoomed in to at least 200% to confirm consistency in shape, size, etc.)

thanks!
